Source/player.h

```diff
@@ -189,8 +189,8 @@ constexpr std::array<char, 6> CharChar = {
 	'r', // rogue
 	's', // sorcerer
 	'm', // monk
-	'b',
-	'c',
+	'b', // Bard
+	'c', // Barbarian

```

Comment:
Just matching casing with all the other comments around it.
```suggestion
	'b', // bard
	'c', // barbarian
```